test/module/irohad/model/converters/json_block_test.cpp

```diff
@@ -28,7 +28,7 @@ class JsonBlockTest : public ::testing::Test {
 };
 
 TEST_F(JsonBlockTest, ValidWhenWellFormed) {
-  Block orig_block;
+  Block orig_block{};
```

Comment:
Why?

Reply:
Default-initializes fields of struct, thanks to missing default constructor.